straxen/plugins/pulse_processing.py

```diff
@@ -263,27 +237,35 @@ class PulseProcessingHighEnergy(PulseProcessing):
     child_plugin = True
     save_when = strax.SaveWhen.TARGET
 
+    n_he_pmts = straxen.URLConfig('', track=False, default=752, infer_type=False,
+                 help="Maximum channel of the he channels")
+
+    record_length = straxen.URLConfig('', default=110, track=False, type=int,
+                 help="Number of samples per raw_record")
+                
+    hit_min_amplitude_he = straxen.URLConfig(
+        default=HITFINDER_DEFAULTS_he['hit_min_amplitude_he'], track=True, infer_type=False,
+        help='Minimum hit amplitude in ADC counts above baseline. '
+                'Specify as a tuple of length n_tpc_pmts, or a number,'
+                'or a string like "pmt_commissioning_initial" which means calling'
+                'hitfinder_thresholds.py'
+                'or a tuple like (correction=str, version=str, nT=boolean),'
+                'which means we are using cmt.'
+        )
+
     def infer_dtype(self):
         dtype = dict()
-        dtype['records_he'] = strax.record_dtype(self.config["record_length"])
-        dtype['pulse_counts_he'] = pulse_count_dtype(self.config['n_he_pmts'])
+        dtype['records_he'] = strax.record_dtype(self.record_length)
+        dtype['pulse_counts_he'] = pulse_count_dtype(self.n_he_pmts)
         return dtype
 
     def setup(self):
         self.hev_enabled = False
+
+        #FIXME: This looks hacky. Maybe find a better way?
```

Comment:
I agree that it looks hacky, now that we are changing the lineage anyway, we can just change this to read len(self.hit_thresholds) right?

Reply:
Leaving this for now, not a priority and scared i may break something.